react/features/analytics/AnalyticsEvents.js

```diff
@@ -97,6 +97,19 @@ export const createAudioOnlyDisableEvent = function() {
     };
 };
 
+/**
+ * Creates an event which indicates that the audio-only mode has been turned
+ * on.
+ *
+ * @returns {Object} The event in a format suitable for sending via
+ * sendAnalytics.
+ */
+export const createAudioOnlyEnableEvent = function() {
```

Comment:
maybe we should combine the enabled / disabled functions into one, which takes a boolean instead.